Alignment/OfflineValidation/scripts/GeometryComparisonPlotter.cc

```diff
@@ -266,7 +288,18 @@ void GeometryComparisonPlotter::MakePlots (vector<TString> x, // axes to combine
                                    _window_width,
                                    _window_height);
         c_global[ic]->Divide(x.size(),y.size());
+        
+        if (_make_profile_plots == "true") {
```

Comment:
Not really. There were already some booleans represented as string and I did not think about it and stuck to it. I changed them now